doc/cross_compilers/linux_cross_compiler.md

```diff
@@ -0,0 +1,91 @@
+# Linux Cross Compiler (GCC)
+
+## Summary
+
+The following describes how to create a cross-compiler for the Bareflank hypervisor on Linux. A script to automate this is provided in the repository.
+
+## Apt
+
+This guide was designed around a bare install of Debian 8.0 (Jessie). Prerequisites for building the cross compiler are satisfied using apt.
+
+[Apt wiki](https://wiki.debian.org/Apt)
+
+You will need to install the following:
+
+```
+apt-get install build-essentials
+apt-get install libmpc
```

Comment:
When you run the configure script for GCC, you can validate if ISL is being used as it will tell you. It's grouped up with the checks for MPC, MPFR, etc...